plugins/LadspaEffect/LadspaEffect.cpp

```diff
@@ -238,11 +228,9 @@ bool LadspaEffect::processAudioBuffer( SampleFrame* _buf,
 				case BufferRate::ControlRateInput:
 					break;
 				case BufferRate::ChannelOut:
-					for( fpp_t frame = 0;
-						frame < frames; ++frame )
+					for (fpp_t frame = 0; frame < outFrames; ++frame)
 					{
-						_buf[frame][channel] = d * _buf[frame][channel] + w * pp->buffer[frame];
-						out_sum += _buf[frame][channel] * _buf[frame][channel];
```

Comment:
For what reason did you move this below?

Reply:
Ladspa plugins may require a lower sample rate, and in those cases, the input buffer is resampled to a new (smaller) frame count. The loop here was calculating the out sum using that smaller buffer then dividing by the smaller frame count later on when calling `checkGate`. However, the `processAudioBuffer` wrapper method doesn't know about this, so it divides the out sum by the actual frame count. I fixed that by calculating the out sum with the actual output buffer like other effects do.